.github/workflows/diff.yml

```diff
@@ -51,15 +53,11 @@ jobs:
           tool: cargo-semver-checks
 
       # if a new line is added here, make sure to update the `summary` job to reference the new step index
-      - uses: taiki-e/install-action@v2
-        with:
-          tool: git-delta
-
-      - run: cargo regress diff --use-pager-directly ${{ matrix.command }}
+      - run: cargo regress diff ${{ matrix.command }}
+      - run: cargo regress diff ${{ matrix.command }}
         env:
           GH_TOKEN: ${{ github.token }}
           GITHUB_PR: ${{ matrix.pr }}
-          GIT_PAGER: delta --hunk-header-style omit
   summary:
```

Comment:
I think we can use `--raw` for this to keep the context delta provides but get rid of the faulty coloring github actions does

Reply:
![image](https://github.com/rust-embedded/svd2rust/assets/1502855/a41ac35b-fcba-4662-985f-4c09e8b7f097)